client/app/bundles/shared/components/Header.scss

```diff
@@ -0,0 +1,46 @@
+@import "base.scss";
+
+.header  {
+width: 100%;
+font-family: $font-family;
+
+  nav {
+    width: 100%;
+    font-family: $font-family;
+    background: $mockup-background-color;
+
+    ul {
+      list-style-type: none;
+      float: right;
+      margin: 0;
+      padding: 0;
+
+      li {
+          display: inline-flex;
+          padding: 10px 5px 10px 20px;
+
+          a {
+            text-decoration: none;
+            color: $header-text-color;
+            font-weight: $lg;
+            line-height: 17px;
+            font-size: $font14;
+            letter-spacing: 2px;
+          }
+        }
+      }
+    }
+
+    .dashboard {
+      color: $dashboard-link-color;
+    }
```

Comment:
Ahh I see what you're getting at here, and this is my fault for not making it clear in the designs, but "active" links (links that reflect what page you're currently on) should have this color, not specifically just the dashboard!

Reply:
Thanks @nshki  for the feedback, I will get on it right away.